corehq/apps/app_manager/templates/app_manager/partials/mobile_ucr_v1_warning.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load xforms_extras %}
+
+  <div class="alert alert-warning alert-build">
+    <h4 class="alert-heading">
+      <i class="fa fa-exclamation-circle"></i>
+      {% trans "Warning while making new version" %}</h4>
```

Comment:
Since we are letting user make a new version, you could consider framing this as
`Warning for the version {version}` or just `Warning` since the message already has the version number.

Saying, `warning while making a new version` is a bit confusing because the version was made and was not blocked which seems right since warnings are non blocking but usually a warning isn't a result of an action, an error is, which this is not. So, just for a better user experience, good to keep that text more clear.

Reply:
I like the suggestion of just `Warning`, it's a bit more concise and better conveys that the new version was still created. Addressed in [e2a069f](https://github.com/dimagi/commcare-hq/pull/35168/commits/e2a069f167cca6709aa463e93e17b71816ac3308).